### PR TITLE
Do not use RFC3986, closes #157

### LINKED ISSUE #157: Do not use RFC3986
It is superceded by https://url.spec.whatwg.org/, which you use extensively. Referencing it causes some confusion due to conflicting normative requirements.

### PR DESCRIPTION
**Pull request by kenchris**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -456,9 +456,9 @@
     <code>null</code>.
   </p>
   <p>
-    The <dfn id="web-nfc-id">Web NFC Id</dfn> is a <a>URL</a> according to
-    [[RFC3986]], specifically the <a>Web NFC message origin</a> optionally
-    followed by a <a>URL path</a>, stored in the <a>Web NFC Record</a>.
+    The <dfn id="web-nfc-id">Web NFC Id</dfn> is an <a>absolute-URL string</a>,
+    specifically the <a>Web NFC message origin</a> optionally
+    followed by an <a>path-absolute-URL string</a>, stored in the <a>Web NFC Record</a>.
```
Comment (anssiko) [inline index.html]: a path

**Review by anssiko — CHANGES_REQUESTED** [review]

**Comment by kenchris** [comment]
This follows the specs and 

screenshot 2018-12-03 at 13 37 22

File: index.html (hunk 1 above)
Comment (kenchris) [reply index.html]: Fixed

**Comment by kenchris** [comment]
PTAL @zolkis

**Comment by zolkis** [comment]
On a quick look, fine for me.

**Pull request merged**